.github/workflows/live-protection.yml

```diff
@@ -0,0 +1,16 @@
+on: [pull_request_target]
+
+jobs:
+  comment:
+    if: ${{ github.event.issue.user.login != 'cxwtool' && github.ref == 'refs/heads/live' }}
```

Comment:
Is there anyway to also check if the PR is coming from `main`, or coming from an admin user? Those two use cases would be helpful to prevent noise.

Reply:
I think even if it comes from an admin user, it's good to notify. See https://github.com/dotnet/docs/pull/21749, which I think may have been opened in `live` by mistake.